internal/database/referendum.go

```diff
@@ -0,0 +1,82 @@
+package database
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+
+	// "database/sql"
+	// "errors"
+	"fmt"
+
+	"github.com/uptrace/bun"
+)
+
+type Referendum struct {
```

Comment:
Consider adding an explicit interface guard near the definition to make it obvious that this should implement `Pollable`, eg:

```go
var _ Pollable = (*Referendum)(nil)
```